lib/src/lifecycle_module.dart

```diff
@@ -951,7 +982,13 @@ abstract class LifecycleModule extends SimpleModule with Disposable {
       await onDidUnloadChildModule(module);
       _didUnloadChildModuleController.add(module);
     } catch (error, stackTrace) {
+      logger.severe(
+        'Exception during onDidUnloadChildModule ($name)',
+        error,
+        stackTrace,
+      );
       _didUnloadChildModuleController.addError(error, stackTrace);
+      return;
```

Comment:
Is this return necessary?

Reply:
Nope, left over from a change I reverted.